cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -7,3 +7,7 @@ options:
     type: string
     default: cluster.local
     description: The local domain for cluster dns
+  service-cidr:
+    type: string
+    default: 10.152.183.0/24
+    description: CIDR to user for Kubernetes services. Cannot be changed after deployment.
```

Comment:
I disagree about layer config, that means, you basically have to compile the charm to change the network CIDR if it conflicts with the network. I'd rather have the hook error saying "You can't change CIDR after deployment"